styleguide/docs/code_conventions.md

```diff
@@ -10,13 +29,73 @@ You can view both the Twig syntax and the rendered HTML markup for a pattern by
 Images should only be committed to the style guide when they are providing "real representative content" in [Pattern Lab "Page" components](http://atomicdesign.bradfrost.com/chapter-2/#pages). Otherwise, dynamic placeholder images should be used. Placeholders should be generated in the form `https://ipsumimage.appspot.com/600x400?l=3:2|600x400&s=36` using the reduced ratio of the image for the benefit of the content team and the rendered dimensions of the image for the benefit of development teams implementing the pattern in a CMS.
 
 Breaking this url down:
+
 - `http://ipsumimage.appspot.com/`: the image generator. More [documentation available there](http://ipsumimage.appspot.com/).
 - `600x400` the width x height with which the placeholder image will be generated
 - `?l=3:2|600x400` the label text, with a pipe (|) separating lines. Our images should use [ratio]|[dimensions].
-- `&s=36` the text size. This should only be tweaked if necessary.\
+- `&s=36` the text size. This should only be tweaked if necessary.
 
 ![This is a placeholder image](https://ipsumimage.appspot.com/600x400?l=3:2|600x400&s=36)
 
+### Pseudo-Patterns
+
+[Pseudo-patterns](http://patternlab.io/docs/pattern-pseudo-patterns.html) are Pattern Lab's way of easily managing closely related patterns that have multiple variants. In the case of our style guide, we typically use pseudo-patterns for patterns where the data structures are consistent, but content might be displayed in differently if certain conditions are met. 
+
+
+Imagine that a CMS serving data to a pattern uses just one data model/content type, but has several different possible ways that content should be displayed. For example:
+* The content must be displayed in different places across the page/site
+* The content must be displayed differently if certain fields are or are not populated
+* The content must be displayed differently depending on whether or not a certain kind of user is logged in
+
+In Drupal, we often use *view modes* to display data in cases like these, so building different view modes in Drupal can often be a good case for using pseudo-patterns in the Style Guide. There are other possible use cases for pseudo-patterns, such as showing pattern states, but this is just the most common example.
```

Comment:
I'd recommend "display/view modes" and a link to https://www.drupal.org/docs/8/api/entity-api/display-modes-view-modes-and-form-modes